src/main/tools/build-runfiles.cc

```diff
@@ -117,6 +117,10 @@ class RunfilesCreator {
 
   void ReadManifest(const std::string &manifest_file, bool allow_relative,
                     bool use_metadata) {
+    if (unlink(temp_filename_.c_str()) != 0 && errno != ENOENT) {
```

Comment:
Please add a comment before this line explaining the purpose of the `unlink` (e.g. "Remove read-only file left over from previous invocation.")

Reply:
Done!